include/vcpkg/base/messages.h

```diff
@@ -733,14 +748,20 @@ namespace vcpkg
                     (msg::path),
                     "",
                     "Both a manifest file and a CONTROL file exist in port directory: {path}");
+    DECLARE_MESSAGE(CopyingHeaders, (msg::path), "", "Copying headers...\nFrom {path}\nTo ");
+    DECLARE_MESSAGE(CopyingLibs, (msg::path), "", "Copying libs...\nFrom {path}\nTo ");
     DECLARE_MESSAGE(CopyrightIsDir, (msg::path), "", "`{path}` being a directory is deprecated.");
     DECLARE_MESSAGE(CorruptedDatabase, (), "", "Database corrupted.");
     DECLARE_MESSAGE(CouldNotDeduceNugetIdAndVersion,
                     (msg::path),
                     "",
                     "Could not deduce nuget id and version from filename: {path}");
-    DECLARE_MESSAGE(CreateFailureLogsDir, (msg::path), "", "Creating failure logs output directory {path}.");
     DECLARE_MESSAGE(CreatedNuGetPackage, (msg::path), "", "Created nupkg: \"{path}\"");
```

Comment:
I am not sure what you mean. Can you elaborate?

Reply:
We have a bunch of different patterns here, with different error message strategies, etc:

```
* Creating <type>...
* created <path>.
and
* Creating <type> <path>...
* done.
and
* (nothing)
* Created <type> <path>.
and
* Creating <type>
* Created <type> <path>.
```

I'm saying I think we should change all such patterns to be the same. My suggestion is:

```
* Creating <type> <path>...
```

with no "done" message. (The "done" is implied by the next "Creating" message or by the program terminating)

This reduces the number of localized messages we need per <type> to 1. So you'd have:

```
DECLARE_MESSAGE(CreatingNuGetPackage, (msg::path), "", "Creating nupkg \"{path}\"...");
```

Reply:
I see; in that case, is it OK to hold off on this kind of refactoring until all the PRs are merged? That way, I can avoid merge conflicts as much as possible and can look at all the messages together :)

Reply:
Yes, that's what I meant by:

>I would really like to see the consistent "creating {path}" treatment done here as even this one command isn't consistent about where it puts and does not put paths, but I would understand if you were like "Bill, this is just trying to localize what we have, not trying to fix the world"

The argument to do it now is that it avoids wasted effort on the part of our localizers because half the reason to choose this form is to reduce the total number of localized messages.